app/views/businessmatching/updateservice/add/msb_subservices.scala.html

```diff
@@ -45,30 +45,30 @@
       ) {
           @checkbox(
             f = f("msbServices[]"),
-            labelText = "businessmatching.updateservice.msb.services.list.lbl.01",
+            labelText = "businessmatching.updateservice.msb.services.list.lbl.03",
```

Comment:
Is this a safe change? Although we have changed the label text the page will still pass the value `01` through to the controller and model. If "01" translates to "Transmitting Money" then there is a disconnect between label text and value?

Reply:
Swapped numbers around in the messages file rather than the view so all match value